src/R/Components/Impl/Application/Configuration/IO/ConfigurationSettingsWriter.cs

```diff
@@ -29,16 +30,18 @@ public sealed class ConfigurationSettingsWriter : IConfigurationSettingsWriter {
 
         /// <summary>
         /// Persists settings into R file. Settings are written
-        /// into R file as assignment statements such as 'name &lt;- value'.
+        /// into R file as assignment statements such as 'settings$name &lt;- value'.
         /// Value
         /// </summary>
         public void SaveSettings(IEnumerable<IConfigurationSetting> settings) {
             WriteHeader();
+            _writer.WriteLine("settings <- new.env()");
```

Comment:
I had considered this, but it was a much larger change to the parsing code, and I wasn't sure if it was beneficial.

One thing that's not shown in my example at the top is that each setting may have metadata associated with it, which is stored in comments right now,  So with the new format it would look like this:
```
settings <- as.environment(list(
    # [Category] SQL
    # [Description] Database connection string
    # [Editor] ConnectionStringEditor
    name1 = 'value1',

    # [Category] SQL
    # [Description] Database connection string
    # [Editor] ConnectionStringEditor
    name2 = 'value2'));
```

One downside with that format is we would lose the ability to have a setting that references another. We have 2 types of setting right now, string or expression.
For example, right now it's possible to have:
```
first_name = 'john'
last_name = 'doe'
full_name = first_name + last_name
```